openedx/adg/admin_dashboard/applications/constants.py

```diff
@@ -16,16 +16,19 @@
     UserApplication.OPEN: {
         'title': _('Open Applications'),
         'description': _('Applications that still need to be reviewed are listed below.'),
-        'style_class': 'open-applications'
+        'style_class': 'open-applications',
+        'no_data_error_msg': _('You don\'t have any application to review')
```

Comment:
Pls add a fullstop at the end of the sentence also I think `do not` sounds better than `don't`, WDYT?

Reply:
Also please add trailing comma as in future if someone adds something else then it will not show up in git diff.